conduct.md

```diff
@@ -4,66 +4,85 @@ layout: page
 
 ## Code of Conduct
 
-DjangoCon Europe aims to be a welcoming event where people meet in a friendly environment.
-Accordingly, all participants are expected to show respect and courtesy to other participants
-throughout the conference.
+All delegates/attendees, speakers, exhibitors, organizers and
+volunteers at DjangoCon Europe 2018 are required to conform to the following Code of Conduct. Organizers will enforce this code before and throughout the event (including related activities such as the social events).
```

Comment:
"They will introduce a Code of Conduct team that will be primarily responsible and will follow the procedures listed below.", maybe, to make clear that this will not only be the organizers in general, but that there will be a dedicated point of contact.

Reply:
👍